draft-ietf-ohai-svcb-config.md

```diff
@@ -238,10 +238,11 @@ for more discussion.
 
 # Gateway Location {#gateway-location}
 
-Clients that know a service is available as a target
-via discovery through the "ohttp" parameter in a SVCB or HTTPS
-record need to know the location of the associated gateway
-before sending oblivious requests.
+Once a client has discovered that a service supports Oblivious HTTP
+via the "ohttp" parameter in a SVCB or HTTPS record, it needs to be
+able to send requests via a relay to the correct gateway location.
+A relay needs to be able to learn location of the gateway associated
+with a service based on the service name alone.
```

Comment:
I'm going to suggest just removing this last sentence — I don't think it adds value beyond the first, and is adding more confusion.

```suggestion
```